spec/factories/reminders.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :reminder do
-    full_name { "Miss Sophia Xu" }
-    email_address { "s.xu@hotmail.com" }
+    full_name { Faker::Name.name }
```

Comment:
@d-a-v-e I think if we are going to introduce Faker for data then we should be moving to using it all over the RSpec suite. Personally I feel we should probably not introduce that at this point. I think the original team made a conscious decision not to use this gem in order to keep the number of gem dependencies to a minimum. Was there a particular reason for deciding to add the gem, that mocked data that existed before did not fill?

Reply:
I created a few sets/list of reminder emails and I didn't want duplicate records in the DB so faker seemed like a good choice. I'm happy to leave it for now.

Reply:
@d-a-v-e ok makes sense